### ISSUE TITLE
Accessibility: pages should have proper labels in icon view

### ISSUE THREAD

**Issue by vvillenave** [pdfarranger/pdfarranger#514]
As a vanilla GTK application, PDF Arranger is quite accessible, except  
for one annoying detail that makes it very hard to use for visually-impaired  
users: the pages are only rendered through the at-spi bridge as unnamed icons.

- Launch a screen reader (for example Orca under GNU/Linux).
- As soon as you hear “Screen reader enabled”, open PDF Arranger  
(any version, master’s fine).
- Hit F10 and navigate through the menus: you can hear every command  
and item properly vocalized.
- Import a multi-page PDF document, and hit Tab until you get to the icon view.
- Move through the pages using the arrow keys: from that point on, the only  
thing you’re hearing is: “icon”, “icon”, “icon”.

Ideally, what you hear should be every page’s label, as printed on the screen.  
This becomes particularly annoying when you mix up the page order (since the  
page numbers won’t be rendered audibly) or when you splice two documents  
together (since nothing differentiates the pages from either one of the sources).

You can test the at-spi tree either through a screen reader or through [Accerciser](https://wiki.gnome.org/Apps/Accerciser).  
I’m personally able and willing to give a hand, since PDF Arranger is such a nifty  
and useful toolbox!

**Comment by dreua** [comment]
Hopefully this is an easy fix on our side and not some nasty PyGtk Bug. Thanks for the report, PRs welcome!

**Comment by jeromerobert** [comment]
Those "icon" is probably the role name of the icon. This is what we are using in tests: <https://github.com/pdfarranger/pdfarranger/blob/6ca5574/tests/test.py#L162>

We should probably use https://lazka.github.io/pgi-docs/index.html#Atk-1.0/classes/Object.html#Atk.Object.set_description or `set_name` to give the right name to the icon.

Unfortunnalty I have no idea on how to get access to the widget representing the Iconview cells. All what we have are 2 `GtkCellRenderer`, and one for the thumbnails and one for the text. I would have expected AT-SPI to use the text GtkCellRenderer automatically.

Still investigating...

**Comment by vvillenave** [comment]
I can confirm that the fix works. Thank you so much you guys!
